src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
+
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo,
+  handelCloseModal: () => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  handelCloseModal,
+}) => {
+  const [user, setUser] = useState<User>();
+  const [isLoadingModal, setIsLoadingModal] = useState(true);
+
+  useEffect(() => {
+    getUser(selectedTodo.userId)
+      .then(setUser)
+      .finally(() => setIsLoadingModal(false));
+  }, []);
```

Comment:
<img width="465" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-todos/assets/49764881/8979ab5b-99d5-4966-a80b-7336f3c05c98">